index.yaml

```diff
@@ -101,6 +101,19 @@ indexes:
   - name: last_updated
     direction: desc
 
+- kind: FeedbackMessageModel
```

Comment:
I'm not sure if these were generated automatically when you had queries last time. If they were, and you don't need these indexes any more, could you please remove them?